media/system/js/fields/fielduser.js

```diff
@@ -6,80 +6,82 @@
-		this.$modal.on('hide', this.removeIframe.bind(this));
+		this.buttonSelect.addEventListener('click', this.modalOpen.bind(this));
+		this.modal.addEventListener('hide', this.removeIframe.bind(this));
 
 		// Check for onchange callback,
-		var onchangeStr =  this.$input.attr('data-onchange'), onchangeCallback;
+		var onchangeStr =  this.input.setAttribute('data-onchange'), onchangeCallback;
 		if(onchangeStr) {
 			onchangeCallback = new Function(onchangeStr);
-			this.$input.on('change', onchangeCallback.bind(this.$input));
+			this.input.addEventListener('change', onchangeCallback.bind(this.input));
 		}
 
 	};
 
 	// display modal for select the file
-	$.fieldUser.prototype.modalOpen = function() {
-		var $iframe = $('<iframe>', {
+	Joomla.fieldUser.prototype.modalOpen = function() {
+		var iframe = document.getElementById('<iframe>', {
 			name: 'field-user-modal',
-			src: this.options.url.replace('{field-user-id}', this.$input.attr('id')),
+			src: this.options.url.replace('{field-user-id}', this.input.attr('id')),
 			width: this.options.modalWidth,
 			height: this.options.modalHeight
 		});
-		this.$modalBody.append($iframe);
-		this.$modal.modal('show');
-		$('body').addClass('modal-open');
+		this.modalBody.append(iframe);
+		this.modal.modal('show');
+		document.getElementsByTagName('body').add('modal-open');
 
 		var self = this; // save context
-		$iframe.load(function(){
-			var content = $(this).contents();
+		iframe.onload(function(){
+			var content = event.target.contents();
 
 			// handle value select
-			content.on('click', '.button-select', function(){
-				self.setValue($(this).data('user-value'), $(this).data('user-name'));
+			content.addEventListener('click', '.button-select', function(){
+				self.setValue(event.target.data('user-value'), event.target.data('user-name'));
 				self.modalClose();
-				$('body').removeClass('modal-open');
+				document.getElementsByTagName('body').classList.remove('modal-open');
 			});
 		});
 	};
 
 	// close modal
-	$.fieldUser.prototype.modalClose = function() {
-		this.$modal.modal('hide');
-		this.$modalBody.empty();
-		$('body').removeClass('modal-open');
+	Joomla.fieldUser.prototype.modalClose = function() {
+		this.modal.modal('hide');
+		this.modalBody.empty();
+		document.getElementsByTagName('body').classList.remove('modal-open');
 	};
 
 	// close modal
-	$.fieldUser.prototype.removeIframe = function() {
-		this.$modalBody.empty();
-		$('body').removeClass('modal-open');
+	Joomla.fieldUser.prototype.removeIframe = function() {
+		this.modalBody.empty();
+		document.getElementsByTagName('body').classList.remove('modal-open');
 	};
 
 	// set the value
-	$.fieldUser.prototype.setValue = function(value, name) {
-		this.$input.val(value).trigger('change');
-		this.$inputName.val(name || value).trigger('change');
+	Joomla.fieldUser.prototype.setValue = function(value, name) {
+		this.input.val(value).trigger('change');
```

Comment:
AFAIK these jQuery methods are not available in plain JS as they are. So these `trigger()` wont probably work